File: hash.java
import java.util.HashMap;

public class Main {
    public static void main(String[] args) {
        HashMap<String, Integer> hashTable = new HashMap<>();
        hashTable.put("apple", 3);
        hashTable.put("banana", 5);
        System.out.println(hashTable.get("banana")); // 5
    }
}

File: hash.cpp
#include <iostream>
#include <unordered_map>
using namespace std;

int main() {
    unordered_map<string, int> hashTable;
    hashTable["apple"] = 3;
    hashTable["banana"] = 5;
    cout << hashTable["banana"] << endl; // 5
}